### ISSUE TITLE
Can we remove the parameter server implementation with Python

### ISSUE THREAD

**Issue by Yancey0623** [sql-machine-learning/elasticdl#2196]
There is two entry point of the parameter server instance, the  one is Python [/elasticdl/python/ps](https://github.com/sql-machine-learning/elasticdl/blob/develop/elasticdl/python/ps/main.py), another one is [/elasticdl/go/cmd/elasticdl_ps](https://github.com/sql-machine-learning/elasticdl/blob/develop/elasticdl/go/cmd/elasticdl_ps/main.go). 

It seems the Python code is useless, can we remove it from this repo?
c.f.
https://github.com/sql-machine-learning/elasticdl/blob/6b1a1b2cd59d07bba642ff648afc0ad42160ee16/elasticdl/python/master/master.py#L414

**Comment by QiJune** [comment]
Yes, we could remove the parameter server implementation with Python. 

The parameter server implementation with Go has been used in production already. It's safe to remove Python codes.
